src/main/java/jenkins/benchmark/jmh/JmhBenchmarkState.java

```diff
@@ -1,6 +1,6 @@
 package jenkins.benchmark.jmh;
 
-import hudson.model.Hudson;
+import hudson.model.*;       //syntax warning fix
```

Comment:
Please do not use wildcard imports.

If you wish to suppress a deprecation warning, best to actually fix the usage, but if that is impossible, use a FQN rather than an `import` statement, and use

```java
@SuppressWarnings("deprecation")
```

in the smallest possible enclosing scope.